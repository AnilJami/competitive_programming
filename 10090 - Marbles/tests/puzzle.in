TRGSJ
XDOKI
M VLN
WPABE
UQHCF
ARRBBL0
ABCDE
FGHIJ
KLMNO
PQRS 
TUVWX
AAA
LLLL0
ABCDE
FGHIJ
KLMNO
PQRS 
TUVWX
AAAAABBRRRLL0
BPJEQ
TAMRG
UHLDS
CFKVX
 NOIW
AARBRR0
MIHLK
ANQDJ
 RGCS
UEFBT
WXOPV
AABRRRRBLA0
MIHLK
ANQDJ
 RGCS
UEFBT
WXOPV
AARRBBRBRBRAA0
Z
