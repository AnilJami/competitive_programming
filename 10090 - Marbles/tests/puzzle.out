Puzzle #1:
T R G S J
X O K L I
M D V B N
W P   A E
U Q H C F

Puzzle #2:
  A B C D
F G H I E
K L M N J
P Q R S O
T U V W X

Puzzle #3:
This puzzle has no final configuration.

Puzzle #4:
B P J E Q
T A M R G
H F L D S
U K V   X
C N O I W

Puzzle #5:
M I H L K
N Q D   S
A R G J C
U E F B T
W X O P V

Puzzle #6
This puzzle has no final configuration.